### DISCUSSION TITLE
Getting error "Cannot find module 'gatsby-core-utils/is-truthy'" when running gatsby new / gatsby develop command

### DISCUSSION THREAD

**Discussion by multibdoyle** [gatsbyjs/gatsby#37113] [Help]
I am working on Gatsby again after about a year of not developing and have received this error, most recently when trying to install the [Gatsby Starter blog](https://www.gatsbyjs.com/starters/gatsbyjs/gatsby-starter-blog):

Cannot find module 'gatsby-core-utils/is-truthy'
Require stack:
- /Users/brendandoyle/.nvm/versions/node/v12.13.1/lib/node_modules/gatsby-cli/lib/reporter/reporter.js
- /Users/brendandoyle/.nvm/versions/node/v12.13.1/lib/node_modules/gatsby-cli/lib/reporter/catch-exit-signals.js
- /Users/brendandoyle/.nvm/versions/node/v12.13.1/lib/node_modules/gatsby-cli/lib/reporter/index.js
- /Users/brendandoyle/.nvm/versions/node/v12.13.1/lib/node_modules/gatsby-cli/lib/create-cli.js
- /Users/brendandoyle/.nvm/versions/node/v12.13.1/lib/node_modules/gatsby-cli/lib/index.js
- /Users/brendandoyle/.nvm/versions/node/v12.13.1/lib/node_modules/gatsby-cli/cli.js

I am using Node Version manager v.14.15.0, Gatsby version 3.9.0 and gatsby-cli version 4.23.0.

Any help would be much appreciated. 

Brendan

**Comment by LekoArts** [comment]
With Gatsby 5 you have to use Node 18. So install and use that with Gatsby. Also, depending on if you've used the global `gatsby-cli` or used the `npx gatsby new` command you'll need to update your `gatsby-cli` installation.
